### ISSUE TITLE
XML tags in JavaDoc comments may not work

### ISSUE THREAD

**Issue by mhdirkse** [frankframework/frank-doc#118]
In Constants.java, I was updating the documentation of the `<Module>` tag. In the text, I added the word `<Module>`, but that word was omitted from the text rendered in the Frank!Doc webpage. I can investigate whether this issue applies to JavaDoc comments in the F!F code as well.

**Comment by mhdirkse** [comment]
Update: We should tag that escaped HTML is rendered correctly. We should check that valid HTML that is not escaped is used to mark-up the text.

**Comment by mhdirkse** [comment]
I tested whether this is really an issue and I think not. To test this, I added a class DummyPipe that implements IPipe in the F!F.

I viewed the description of an attribute in Eclipse:

![image](https://user-images.githubusercontent.com/6784732/169824490-7e12e0c1-aa14-4075-b88b-e004dfbcbfa1.png)

This looks OK. There is a word "name" between <code></code> that still appears. There is a word "element" between &lt; and &gt; that appears as `<element>`. The latter does not work in VSCode, but that is not a Frank!Doc issue.

The same DummyPipe appears in the Frank!Doc webapp as follows:

![image](https://user-images.githubusercontent.com/6784732/169824849-4797c5c5-72d3-4e13-9673-7d09832c3b34.png)

This looks OK.
